types/node/index.d.ts

```diff
@@ -2225,24 +2225,31 @@ declare module "child_process" {
         encoding: string | null; // specify `null`.
     }
 
+    export interface ExecException extends Error {
+        cmd?: string;
```

Comment:
Are these all optional?

Reply:
No, good point. I was not 100% sure, so went with optional (which in hindsight might not have been the best idea).

Looking at [the source](https://github.com/nodejs/node/blob/5842366ae83a36065d626e3937ad8fc327efab30/lib/child_process.js#L286-L294) it looks like `cmd` is not optional - the others are.